module-2/backend/week_three.md

```diff
@@ -10,14 +10,54 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+
+rails new NAME -T --database=postgresql --skip-spring --skip-turbolinks
+
 2. What do Models generally inherit from in rails?
+
+Rails' ApplicationRecord
+
 3. What do Controllers generally inherit from in a rails project?
+
+Rails' ApplicationController
+
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+
+resources :horse, only [:show]
+
 5. What rake task is useful when looking at routes, and what information does it give you?
+
+rake routes, this gives you the action, path, and URL
+
 6. What is an example of a route helper? When would you use them?
+
+A route helper is something like `link_to` where you can pass it ActiveRecord objects.
```

Comment:
horses_path is a route helper and i would use it in the view/controller/test as a reference to a path